src/autocomplete.js

```diff
@@ -50,7 +50,8 @@ export default class Autocomplete extends Component {
     required: false,
     tNoResults: () => 'No results found',
     tAssistiveHint: () => 'When autocomplete results are available use up and down arrows to review and enter to select.  Touch device users, explore by touch or with swipe gestures.',
-    dropdownArrow: DropdownArrowDown
+    dropdownArrow: DropdownArrowDown,
+    ariaLabelledBy: undefined | ''
```

Comment:
Is this meant to be an OR (`||`)?

Reply:
@JakeChampion Yes, you're right. I've updated it.